content/blog/similar-document-search-using-apache-spark-with-tf-idf.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Similar Document Search using Apache Spark with TF-IDF
+date: 2021-06-15T06:09:32.919Z
+author: Prasad Singathi, Maikel Pereira
+authorimage: /img/maikel-pereira.jpg
+tags:
+  - MapR
+  - HPE Ezmeral Data Fabric
+  - Machine learning
+---
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/data-fabric.html>
+
+## Original Post Information:
+
+```
```

Comment:
Make sure you specify a language syntax for each and every code block snippet.
You can do it easily either from "Rich Text" mode (there is a "settings" wheel icon on the right side of the code block, or use "markdown" mode in the editor, and specify a language syntax.

I recommend syntax language "markdown" for this code block.